src/utils/gitUtil.mts

```diff
@@ -7,9 +7,29 @@ import { SettingsKey, HOME_VAR } from "../settings.mjs";
 import { homedir } from "os";
 import which from "which";
 import { window } from "vscode";
+import { compareGe } from "./semverUtil.mjs";
 
 export const execAsync = promisify(exec);
 
+export const MIN_GIT_VERSION="2.28.0";
+
+export async function checkGitVersion(gitExecutable: string):
+  Promise<[boolean, string]> 
+{
+  const versionCommand =
+    `${
+      process.env.ComSpec === "powershell.exe" ? "&" : ""
+    }"${gitExecutable}" version`;
+  const ret = await execAsync(versionCommand)
+  const stdout = ret.stdout.split(" ");
+  let gitVersion = stdout[stdout.length-1];
+  if (gitVersion.includes("windows")) {
+    gitVersion = gitVersion.split(".windows")[0];
+  }
```

Comment:
This already accounts for the special naming on Windows but on macOS and any other custom build there are different suffixes. It seems like the prefix format `git version <some semver>` is always the same. Therefore I would get the third item (if len >= 3) or with this regex:
```regex
/git version (\d+\.\d+(\.\d+)*)/g
```

Reply:
That regex works nicely - thanks